xds/internal/client/client_requests_counter.go

```diff
@@ -0,0 +1,88 @@
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package client
+
+import (
+	"fmt"
+	"sync"
+	"sync/atomic"
+)
+
+type servicesRequestsCounter struct {
+	mu       sync.Mutex
+	services map[string]*ServiceRequestsCounter
+}
+
+var src = &servicesRequestsCounter{
+	services: make(map[string]*ServiceRequestsCounter),
+}
+
+// ServiceRequestsCounter is used to track the total inflight requests for a
+// service with the provided name.
+type ServiceRequestsCounter struct {
+	ServiceName string
+	maxRequests uint32
+	numRequests uint32
+}
+
+// GetServiceRequestsCounter returns the ServiceRequestsCounter with the
+// provided serviceName. If one does not exist, it creates it.
+func GetServiceRequestsCounter(serviceName string) *ServiceRequestsCounter {
+	src.mu.Lock()
+	defer src.mu.Unlock()
+	c, ok := src.services[serviceName]
+	if !ok {
+		c = &ServiceRequestsCounter{ServiceName: serviceName, maxRequests: 1024}
```

Comment:
Define a `const` for `1024`, and use the const here and following.

Reply:
Made this change.